src/cli/main.cc

```diff
@@ -52,6 +52,18 @@ extern "C" void SignalHandler(int sig) {
   }
 }
 
+extern "C" void SigUsr1lHandler(int sig) {
```

Comment:
It seems weird that we use a custom signal to trigger the ingest operation. Maybe we can consider to introduce a new command.